README.md

```diff
@@ -8,6 +8,15 @@ PyMC3 educational resources, including the PyMC3 port of the following books (or
 -  [PyMC3 port of the book "Bayesian Statistical Methods" by Brian J. Reich and Sujit K. Ghosh](https://github.com/pymc-devs/resources/tree/main/BSM)
 -  [PyMC3 port of the book "Bayesian Data Analysis" by Andrew Gelman, John B. Carlin, Hal S. Stern, David B. Dunson, Aki Vehtari and Donald B. Rubin](https://github.com/pymc-devs/resources/tree/main/BDA3)
 
+## How to contribute
+Thanks wanting to contribute! These resources are a community effort and we, and all future resource users, appreciate your help.
+
+If just starting
+1. Reading the [contributing guide for pymc](https://github.com/pymc-devs/pymc/blob/main/CONTRIBUTING.md) is a good place to start. The guide will familiarize you with the high level tools and workflow.
+  * Some of the instructions will differ so read the below steps first.
+2. In this repo the environments are defined per resource. Look into each directory to find the environment file and use that
+3. When ready to contribute open a draft PR stating the scope of work as early as possible. This helps avoid duplicate work early.
+4. If you have further questions feel free to ask on https://discourse.pymc.io/or open an issue.
```

Comment:
```suggestion
4. If you have further questions don't hesitate to ask on https://discourse.pymc.io/.
```

the removing of issue suggestion might be a bit much. I think I haven't even contributed to resources yet and have no idea what dynamics are going on.